### PR TITLE
Patch port with connected port

### PR DESCRIPTION
**Pull request by shivacharanms**
This PR contains the following:
Allow patch on fabric switch ports to include connected ports of system ethernet interfaces

### CODE REVIEW

File: caphandler/ports.go [hunk 1]
```diff
@@ -90,6 +91,67 @@ func GetPortInfo(ctx iris.Context) {
...
+				}
+				if !checkFlag {
+					errMsg := fmt.Sprintf("Ethernet data for uri %s not found", uri)
+					log.Error(errMsg)
+					resp := updateErrorResponse(response.ResourceNotFound, errMsg, []interface{}{"Ethernet", uri})
+					ctx.StatusCode(http.StatusNotFound)
+					ctx.JSON(resp)
+					return
+				}
+				//Check on ODIM if ehternet is valid
```
Comment (jeevan-kamkar) [inline caphandler/ports.go:140]: correct the typo

File: caphandler/ports.go [hunk 2]
```diff
@@ -90,6 +91,67 @@ func GetPortInfo(ctx iris.Context) {
...
+		return
+	}
+	checkFlag := false
+
+	if port.Links != nil {
+		if port.Links.ConnectedPorts != nil {
+			if len(port.Links.ConnectedPorts) > 0 {
+				//Assuming we have only one connected port
+				ethernetURI := port.Links.ConnectedPorts[0].Oid
+				//Check on ODIM if ehternet is valid
```
Comment (jeevan-kamkar) [inline caphandler/ports.go:119]: correct the typo

**Review by jeevan-kamkar — COMMENTED** [review]
correct the typos

**Review by jeevan-kamkar — COMMENTED** [review]
Helm charats need to be updated with the latest configuration https://github.com/ODIM-Project/PluginCiscoACI/blob/development/install/Kubernetes/helmcharts/aciplugin/templates/configmaps.yaml

**Comment by shivacharanms** [comment]
During the sync of the base development branch of the forked repo, this pr was merged. Raising another PR to the main repo with the comments given here.

**Pull request merged**